pyiron_atomistics/lammps/control.py

```diff
@@ -525,7 +525,8 @@ def calc_md(
             if langevin:
                 warnings.warn("Temperature not set; Langevin ignored.")
             fix_ensemble_str = "all nve"
-            initial_temperature = 0
+            if initial_temperature is None:
+                initial_temperature = 0
```

Comment:
```suggestion
            initial_temperature = 0 if initial_temperature is None else initial_temperature
```

I've been one-lining this type of check.